immersive-web-wg-charter.html

```diff
@@ -197,39 +195,36 @@ <h2>
           Scope
         </h2>
         <p>
-          The WebVR Working Group will develop standardized APIs to provide
+          The Immersive Web Working Group will develop standardized APIs to provide
           access to input and output capabilities commonly associated with
-          virtual reality device hardware such as Google’s Daydream, the Oculus Rift,
-          the Samsung Gear VR, the HTC Vive, and Windows Mixed Reality
-          headsets and sensors. In other words, the WG will develop an API to enable the
-          creation of virtual reality web sites that one can view in a VR
-          headset. This VR experience must be embeddable in the Web of today to
+          XR hardware such as Google’s Daydream, the Oculus Rift,
+          the Samsung GearVR, the HTC Vive, and Windows Mixed Reality
+          headsets and sensors as well as mobile handheld devices in "magic window" mode.
+          In other words, the WG will develop an API to enable the creation of XR web sites.
+          This XR experience must be embeddable in the Web of today to
           allow progressive enhancement of existing web sites.
         </p>
         <p>
-          The <strong>scope</strong> of the WebVR Working Group charter is to
+          The <strong>scope</strong> of the Immersive Web Working Group charter is to
           define APIs which:
         </p>
         <ul>
-          <li>Enable a Web page to detect available virtual reality devices.
+          <li>Enable a Web page to detect available XR devices and sensors.
           </li>
-          <li>Allow querying a virtual reality device for device-specific
-          capabilities.
+          <li>Allow querying a XR device for device-specific capabilities.
           </li>
-          <li>Receive updated information about the device's position and
-          orientation over time.
+          <li>Receive updated information about the device's position and orientation over time.
           </li>
-          <li>Present imagery to the device at the device native frame rate.
+          <li>Receive updated information about the device's environment.
+          </li>
+          <li>Present imagery to the device at the device's native frame rate.
           </li>
```

Comment:
I think we need something in here to indicate that basic access to XR input methods (controllers, gestures, etc) is also in scope. (We don't want to imply deep support, though, like full hand tracking. Not sure how best to present that.)

Reply:
I'll take a first pass.